files/es/games/techniques/webrtc_data_channels/index.html

```diff
@@ -27,59 +27,8 @@ <h2 id="Usando_la_libreria_p2p">Usando la libreria p2p</h2>
 <p><strong>Nota:</strong> Continuaremos agregando contenido aquí pronto; hay algunos problemas de organización por resolver.</p>
 </div>
 
-<h2 id="Compatibilidad_de_navegadores">Compatibilidad de navegadores</h2>
 
-<p>{{ CompatibilityTable() }}</p>
 
```

Comment:
Here the script didn't add the replacement with the `{{Compat}}` macro. Is this intended?

Reply:
Cool. Thanks @cw118!